server/janitor/janitor.go

```diff
@@ -1,79 +1,180 @@
 package janitor
 
 import (
+	"context"
 	"flag"
+	"strings"
 	"time"
 
 	"github.com/buildbuddy-io/buildbuddy/server/environment"
 	"github.com/buildbuddy-io/buildbuddy/server/tables"
+	"github.com/buildbuddy-io/buildbuddy/server/util/db"
 	"github.com/buildbuddy-io/buildbuddy/server/util/log"
 )
 
 var (
+	// Flags shared by both invocation and execution janitor.
+	logDeletionErrors = flag.Bool("log_deletion_errors", false, "If true; log errors when ttl-deleting expired data")
+
+	// Flags for Invocaction Janitor.
 	ttlSeconds = flag.Int("storage.ttl_seconds", 0, "The time, in seconds, to keep invocations before deletion. 0 disables invocation deletion.")
 
-	cleanupBatchSize  = flag.Int("storage.cleanup_batch_size", 10, "How many invocations to delete in each janitor cleanup task")
-	cleanupInterval   = flag.Duration("cleanup_interval", 10*60*time.Second, "How often the janitor cleanup tasks will run")
-	cleanupWorkers    = flag.Int("cleanup_workers", 1, "How many cleanup tasks to run")
-	logDeletionErrors = flag.Bool("log_deletion_errors", false, "If true; log errors when ttl-deleting expired data")
+	cleanupBatchSize = flag.Int("storage.cleanup_batch_size", 10, "How many invocations to delete in each janitor cleanup task")
+	cleanupInterval  = flag.Duration("cleanup_interval", 10*60*time.Second, "How often the janitor cleanup tasks will run")
+	cleanupWorkers   = flag.Int("cleanup_workers", 1, "How many cleanup tasks to run")
+
+	// Flags for Invocaction Janitor.
+	executionTTLSeconds = flag.Int("storage.execution.ttl_seconds", 0, "The time, in seconds, to keep invocations before deletion. 0 disables invocation deletion.")
```

Comment:
Ideally this should be a Duration flag, but I can see why you might want to keep it as-is for consistency.
Maybe we can fix this by introducing a new Duration flag for invocations and deprecating the old one.

Reply:
I changed the one for execution into a duration and leave invocation as it is.